README.md

```diff
@@ -34,146 +38,106 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
 Here's the quickest way to get going with Rook.
 
-### Linux
-
-On a modern Linux host run the following:
-
-1. Download the latest  binaries
-
-    ```bash
-    $ wget https://github.com/rook/rook/releases/download/v0.2.2/rook-v0.2.2-linux-amd64.tar.gz
-    $ tar xvf rook-v0.2.2-linux-amd64.tar.gz
-    ```
-
-2. Start a one node Rook cluster
-
-    ```bash
-    $ ./rookd --data-dir /tmp/rook-test
-    ```
-
-#### Block Storage
-1. In a different shell (in the same path) create a new volume image (10MB)
-
-    ```bash
-    $ ./rook block create --name test --size 10485760
-    ```
-
-2. Mount the block volume and format it
-
-    ```bash
-    sudo ./rook block mount --name test --path /tmp/rook-volume
-    sudo chown $USER:$USER /tmp/rook-volume
-    ```
-
-3. Write and read a file
-
-    ```bash
-    echo "Hello Rook!" > /tmp/rook-volume/hello
-    cat /tmp/rook-volume/hello
-    ```
-
-4. Cleanup
-
-    ```bash
-    sudo ./rook block unmount --path /tmp/rook-volume
-    ```
-
-#### Shared File System
-1. Create a shared file system
-
-    ```bash
-    ./rook filesystem create --name testFS
-    ```
-
-2. Verify the shared file system was created
-
-   ```bash
-   ./rook filesystem ls
-   ```
+### Kubernetes
 
-3. Mount the shared file system from the cluster to your local machine
+This example shows how to build a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
 
-   ```bash
-   ./rook filesystem mount --name testFS --path /tmp/rookFS
-   sudo chown $USER:$USER /tmp/rookFS
-   ```
+#### Prerequisites
 
-4. Write and read a file to the shared file system
+This example requires a running Kubernetes cluster. You will also need to modify the kubelet service to bind mount `/sbin/modprobe` to allow access to `modprobe`. Access to modprobe is necessary for using the rbd volume plugin (<https://github.com/kubernetes/kubernetes/issues/23924>).
```

Comment:
- Two paragraphs in a row start with "This example...". What if this paragraph starts with "First we need a running Kubernetes cluster"
- It still feels like starting kubernetes doesn't belong in our main readme.